extensions/amp-list/0.1/amp-list.js

```diff
@@ -277,16 +293,30 @@ export class AmpList extends AMP.BaseElement {
         this.renderItems_ = null;
       }
     };
-    this.templates_.findAndRenderTemplateArray(this.element, current.items)
-        .then(elements => this.updateBindingsForElements_(elements))
-        .then(elements => this.rendered_(elements))
-        .then(/* onFulfilled */ () => {
-          scheduleNextPass();
-          current.resolver();
-        }, /* onRejected */ () => {
-          scheduleNextPass();
-          current.rejecter();
-        });
+    const onFulfilledCallback = () => {
+      scheduleNextPass();
+      current.resolver();
+    };
+    const onRejectedCallback = () => {
+      scheduleNextPass();
+      current.rejecter();
+    };
+    if (this.ssrTemplateHelper_.isSupported()) {
+      // TODO(alabiaga): should call updatebindings as below?
+      this.templates_.findAndRenderTemplate(
+          this.element, /** @type {!HTML} */ (current.data))
+          .then(element => {
+            console.log(element);
+            this.container_.appendChild(element);
+          })
```

Comment:
Why isn't this using the same `updateBindingsForElements, rendered` pipeline as the old way?